packages/react-components/react-combobox/src/components/Listbox/useListboxStyles.ts

```diff
@@ -18,6 +18,7 @@ const useStyles = makeStyles({
     flexDirection: 'column',
     minWidth: '160px',
     overflowY: 'auto',
+    ...shorthands.outline('1px', 'solid', tokens.colorTransparentStroke),
```

Comment:
Feel free to disregard this since it's outside the scope of this change but I think we can use [makeResetStyles()](https://griffel.js.org/react/api/make-reset-styles) here to minimize class name bloat.

Reply:
I kinda want to go through and do some `makeResetStyles` changes in Combobox/Dropdown overall -- might save that for a different PR though

Reply:
I wrote the initial styles before `makeResetStyles`, and I haven't given it a pass through for splitting out not-super-reusable styles yet.